g/graylog-docker/Dockerfiles/4.2.9_ubi-8.5/config/log4j2.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please add proper attribution if this is being picked up / adapted from an existing source